### PR TITLE
Refactor/nf sv4 acl compat homedir consumer

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: config/services.yaml [hunk 1]
```diff
@@ -136,6 +136,7 @@ services:
     App\Consumer\CreateHomedirConsumer:
         bind:
             Symfony\Bridge\Monolog\Logger $logger: '@monolog.logger.create_homedir'
+            $fsType: "%env(FILESYSTEM_TYPE)%"
```
Comment (mickel1138) [inline config/services.yaml]: I like fully spelled out variable names, so you instantly can see what it's for.

**Comment by fightingtexasaggie** [comment]
similarly, I have a bash script ready that can be used to apply ACL/perms/owners to existing homedirs once they're rsynced to the NetApp.

File: src/Consumer/CreateHomedirConsumer.php [hunk 2]
```diff
@@ -54,6 +64,133 @@ public function __construct(
     * @return boolean True if success, false otherwise.
     */
     public function execute(AMQPMessage $message)
+    {
+        if ('NFS' === $this->filesystemType) {
+            return($this->processNFS($message));
+        } else {
```
Comment (mickel1138) [inline src/Consumer/CreateHomedirConsumer.php]: Not a huge deal, but the other setting could be anything else, but is expected to be "Linux" as set in the .env file.

File: src/Consumer/CreateHomedirConsumer.php [hunk 3]
```diff
@@ -54,6 +64,133 @@ public function __construct(
     * @return boolean True if success, false otherwise.
     */
     public function execute(AMQPMessage $message)
+    {
+        if ('Linux' === $this->filesystemType) {
+            return($this->processNormalFS($message));
+        } else {
```
Comment (mickel1138) [inline src/Consumer/CreateHomedirConsumer.php]: My point was more the second else, should be an `if`?

**Pull request merged**